spec/views/items/_embargo_form.html.erb_spec.rb

```diff
@@ -1,13 +1,16 @@
 require 'spec_helper'
 
 RSpec.describe 'items/_embargo_form.html.erb' do
-  let(:current_user) do
-    double('current_user', is_admin: true)
+  let(:current_user) { mock_user(is_admin?: true) }
+
+  before do
+    allow(view).to receive(:current_user).and_return(current_user)
+    allow(controller).to receive(:current_user).and_return(current_user)
```

Comment:
cancan is injecting controller-level methods (`current_ability`, `can?`, and `cannot?`, among others) that are shared with the view. One future refactor could be to separate our view specs from users + abilities by stubbing `can?` directly, eliminating the need for this stubbing.